pkg/registry/seedmanagement/managedseed/strategy.go

```diff
@@ -89,6 +90,11 @@ func mustIncreaseGeneration(oldManagedSeed, newManagedSeed *seedmanagement.Manag
 		return true
 	}
 
+	// The operation annotation was added with value "renew-kubeconfig"
+	if kutil.HasMetaDataAnnotation(&newManagedSeed.ObjectMeta, v1beta1constants.GardenerOperation, managedseed.GardenerOperationRenewKubeconfig) {
```

Comment:
You need to move the `managedseed.GardenerOperationRenewKubeconfig` constant to the `v1beta1constants` package.